mlflow/environment_variables.py

```diff
@@ -63,9 +63,32 @@ def __repr__(self):
 MLFLOW_GCS_DEFAULT_TIMEOUT = _EnvironmentVariable("MLFLOW_GCS_DEFAULT_TIMEOUT", int, None)
 
 #: Specify whether to disable model logging and loading via mlflowdbfs.
-#: (default: ``False``)
+#: (default: ``)
 _DISABLE_MLFLOWDBFS = _EnvironmentVariable("DISABLE_MLFLOWDBFS", str, "")
```

Comment:
Sorry, we recently made a change to the default value here; it is now `None`.

```suggestion
#: (default: `None`)
_DISABLE_MLFLOWDBFS = _EnvironmentVariable("DISABLE_MLFLOWDBFS", str, None)
```